internal/pkg/describe/webapp.go

```diff
@@ -332,6 +344,46 @@ func (d *WebAppDescriber) stackDescriber(roleARN, region string) (stackDescriber
 	return d.stackDescribers[roleARN], nil
 }
 
+// ServiceArn is the arn of an ECS service.
+type ServiceArn string
+
+// ClusterName returns the cluster name.
+// For example: arn:aws:ecs:us-west-2:1234567890:service/my-project-test-Cluster-9F7Y0RLP60R7/my-project-test-my-app-Service-JSOH5GYBFAIB
+// will return my-project-test-Cluster-9F7Y0RLP60R7
+func (s *ServiceArn) ClusterName() (string, error) {
+	serviceArn := string(*s)
+	parsedArn, err := arn.Parse(serviceArn)
+	if err != nil {
+		return "", err
+	}
+	resources := strings.Split(parsedArn.Resource, "/")
+	return resources[1], nil
+}
+
+// ServiceName returns the service name.
+// For example: arn:aws:ecs:us-west-2:1234567890:service/my-project-test-Cluster-9F7Y0RLP60R7/my-project-test-my-app-Service-JSOH5GYBFAIB
+// will return my-project-test-my-app-Service-JSOH5GYBFAIB
+func (s *ServiceArn) ServiceName() (string, error) {
```

Comment:
nit: the receivers can be value-receivers instead of pointers since the data is not big.